tests/e2e/utils.js

```diff
@@ -0,0 +1,23 @@
+import { browser, $ } from '@wdio/globals';
+
+let extensionId = '';
+export async function getExtensionId() {
+  if (!extensionId) {
+    await browser.url('chrome://extensions');
+
+    const extension = await $('extensions-item:first-child');
+    extensionId = await extension.getAttribute('id');
+  }
+
+  return extensionId;
+}
+
+export async function navigateToPage(page, file = 'index.html') {
+  await browser.url(
+    `chrome-extension://${await getExtensionId()}/pages/${page}/${file}`,
+  );
+}
+
+export function getQAElement(qa) {
```

Comment:
maybe `getByTestId`?

Reply:
The WDIO has the `getElement` API, i think we will never try to query our own DOM structure differently, so I even consider renaming it to `getElement`.

Reply:
I'm good with `getElement`, but maybe we can be more explicit by calling it `getExtensionElement` and `navigateToExtensionPage` ?